style.css

```diff
@@ -436,9 +431,195 @@ textarea {
     margin: 0 1.8rem;
   }
 
-  .box2 h1 {
+  .Greetings h1 {
     font-size: 40px;
     max-width: 300px;
     line-height: 3rem;
   }
 }
+
+@media screen and (min-width: 768px) {
+  .box2 {
+    background-image: url(imgs/Header\ bg.svg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    width: 100%;
+    margin-bottom: 100px;
+  }
```

Comment:
- [ ] Great job so far. However, there few things you need to look into before you move on. The background image for the header section  does not seem to match how it appears on the figma template. The headline section is not properly positioned. It is expected to have the same margin for the both sides. But your design looks different from that design. You should also consider to reduce the margin top of the headline so that there would no be need to scroll before you could see the headline on page load. Kindly check the background image you used for that section fix the issue. If you did not use the background image before, kindly consider using it to avoid layout issues.

###  Your Design
[your-headline.webm](https://user-images.githubusercontent.com/26597432/206658034-64ed0dae-7e67-4e09-bfb0-7a8a656a853d.webm)

### Figma Design
![figma-headline](https://user-images.githubusercontent.com/26597432/206658103-b4d972cc-4580-4034-87d1-099fe7e11d40.png)